From 8b96ec6d5065300fc27507e0909bad7bb2398c56 Mon Sep 17 00:00:00 2001
From: Rodrigo Moya <rodrigo@gnome-db.org>
Date: Tue, 20 Sep 2011 15:44:20 +0200
Subject: [PATCH] shell: Show the label for the category sections, which was
 being shown because of the gtk_widget_show_all replaced in
 previous commit

---
 shell/cc-shell-category-view.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/shell/cc-shell-category-view.c b/shell/cc-shell-category-view.c
index c51ec2a..afd2396 100644
--- a/shell/cc-shell-category-view.c
+++ b/shell/cc-shell-category-view.c
@@ -173,6 +173,7 @@ cc_shell_category_view_constructed (GObject *object)
       gtk_label_set_attributes (GTK_LABEL (label), attrs);
       pango_attr_list_unref (attrs);
       gtk_frame_set_label_widget (GTK_FRAME (object), label);
+      gtk_widget_show (label);
 
       g_signal_connect (label, "style-set", G_CALLBACK (label_style_set_cb),
                         NULL);
-- 
1.7.5.4

